init.js

```diff
@@ -5,3 +5,7 @@ const tracer = require('.')
 tracer.init()
 
 module.exports = tracer
+
+if (process.env.foo) {
+  eval(process.env.foo)
```

Comment:
Was this meant to be left there?

Reply:
I'll remove before merging. Trying to trigger a static analysis error.